tensorflow_addons/seq2seq/sampler.py

```diff
@@ -31,23 +31,30 @@ class Sampler(metaclass=abc.ABCMeta):
 
     Sampler instances are used by `BasicDecoder`. The normal usage of a sampler
     is like below:
+
+    ```python
     sampler = Sampler(init_args)
     (initial_finished, initial_inputs) = sampler.initialize(input_tensors)
-    for time_step in range(time):
-      cell_output, cell_state = cell.call(cell_input, previous_state)
+    cell_input = initial_inputs
+    cell_state = cell.get_initial_state(...)
+    for time_step in range(max_output_length):
```

Comment:
Should we prefer using `tf.range` here?

Reply:
One more thing. Although it doesn't matter as it is in the documentation, we should stick to 4 spaces only to make sure things are consistent

Reply:
I just changed the code to use 2-space indent and tf.range().

Reply:
@ageron apologies. I thought it was indented by 2 space earlier. Sorry, I think it was because of an issue in my editor. Can you please push again with 4 spaces indentation? Again, sorry for the miscommunication

Reply:
I updated to 4 spaces so that we can merge this change. Thanks for pointing this out.